sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -30,12 +30,15 @@ object AkkaGrpcPlugin extends AutoPlugin {
   }
   import autoImport._
 
-  override def projectSettings = configSettings(Compile) ++ configSettings(Test)
+  override def projectSettings = defaultSettings ++ configSettings(Compile) ++ configSettings(Test)
+
+  private def defaultSettings =
+    Seq(
+      akkaGrpcCodeGeneratorSettings := Seq("flat_package"),
+      akkaGrpcCodeGenerators := GeneratorAndSettings(ScalaBothCodeGenerator, akkaGrpcCodeGeneratorSettings.value) :: Nil)
```

Comment:
:+1:

Reply:
Couldn't move the third setting out of the config-scope for some reason, but couldn't figure out why